src/components/Landing.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import {users} from './DummyUsers'
+import Table from './Table';
+import "./Landing.css";
+
+class Landing extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            isLoggedIn : true
+        }
+    }
+    render(){
+        const authContainer = this.state.isLoggedIn ? (
+            <div className="auth-container">
+            <button>Logout</button>
+            <div className="user-profile">
+                <img className="user-avatar" src="./image-victor.jpg" alt="user-avatar" />
+                <p className="user-name">Victor</p>
+            </div>
+        </div>
+        ):(
```

Comment:
Conditional rendering for a part that could potentially grow much bigger in structure. Should make that shorter, may be as suggestion :make a header folder and store header component itself and components associated with it.